.sass-lint.yml

```diff
@@ -35,7 +35,9 @@ rules:
   no-empty-rulesets: 2
   no-extends: 0
   no-ids: 2
-  no-important: 2
+  # https://github.com/nhsuk/nhsuk-frontend/blob/master/packages/core/utilities/_typography.scss
+  # "Utility classes are allowed to use !important;"
+  no-important: 0
   no-invalid-hex: 2
```

Comment:
There was some code already in the codebase that was stopping me from committing without this. It is now the same as the dev branch.